lib/vyasa/medium/playback.ex

```diff
@@ -0,0 +1,17 @@
+  defmodule Vyasa.Medium.Playback do
+    @moduledoc """
+    The Playback struct is the bridge between written and media player contexts.
+    """
+
+    defstruct [:medium, playing?: false, played_at: nil, paused_at: nil, elapsed: 0, current_time: 0]
+
+    def new(%{} = attrs) do
+      %Vyasa.Medium.Playback{
+        medium: attrs.medium,
```

Comment:
playback should be agnostic to the medium, so this attribute should be removed 

in that way, the bridge can control just playback 

the mediums shall sync up with the playback  

events are tied to the media bridge

video and audio players can change the playback (e.g. from within the players, a seek is done)

Reply:
this clarifies that the media bridge shall handle playback and shall contain multiple AV players


+ keeps separate audio and video streams and allows for interoperability